week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -2,89 +2,95 @@
 // Look at the tests and see how you can fix them.
 
 function mood() {
-  let isHappy = true;
+    let isHappy = true;
 
-  if (isHappy) {
-    return "I am happy";
-  } else {
-    return "I am not happy";
-  }
+    if (isHappy != true) {
```

Comment:
this definitely works :) but what happens if you are asked to keep the logic as is `if(isHappy)` ? What would you do? It's also easier to read/understand